modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/Reader.java

```diff
@@ -1356,10 +1349,7 @@ protected boolean isOperationHidden(Method method) {
         if (hidden != null) {
             return true;
         }
-        if (config != null && !Boolean.TRUE.equals(config.isReadAllResources()) && apiOperation == null) {
-            return true;
-        }
-        return false;
+        return config != null && !Boolean.TRUE.equals(config.isReadAllResources()) && apiOperation == null;
```

Comment:
Not sure if this is really more readable than the previous version. 🤔

Reply:
I have restored the previous statements.